plugin/type_uuid/item_uuidfunc.h

```diff
@@ -64,4 +64,34 @@ class Item_func_uuid: public Item_func_sys_guid
   { return get_item_copy<Item_func_uuid>(thd, this); }
 };
 
+class Item_func_uuid_v7: public Item_str_func
+{
```

Comment:
Same comment with Iten_func_uuid_v4: please derive Item_func_uuid_v7 from Item_func_sys_guid to avoid overriding too many methods.